components/blocks/About.vue

```diff
@@ -107,6 +117,24 @@ export default {
       required: true,
     },
   },
+  data: () => ({
+    opportunities: [],
+  }),
+  async fetch() {
+    this.opportunities = await fetch(
+      'https://brown.wd5.myworkdayjobs.com/wday/cxs/brown/staff-careers-brown/jobs',
+      {
+        method: 'post',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({
+          limit: 20,
+          offset: 0,
+          appliedFacets: {},
+          searchText: '180 George Street',
+        }),
+      }
+    ).then((res) => res.json());
```

Comment:
```suggestion
    ).then((res) => {
      const data = res.json()
      return data.jobPostings.filter(j -> j.locationsText === '180 George Street')
    });
```
It's a bit odd to mix/match `async await` and `.then`s.  I'd drop the `.then` (the suggestion of data manipulation changes still applies, but can be translated to the top level) and just use awaits where needed.


It might also be worth pulling `180 George Street` into a variable in fetch since we reference it twice.